sickbeard/name_parser/parser.py

```diff
@@ -22,37 +22,44 @@
 import time
 import re
 import os.path
+
 import sickbeard
 from sickbeard.name_parser import regexes
 
 from sickbeard import logger, helpers, scene_numbering, common, scene_exceptions, db
+from sickbeard.name_parser.guessit_parser import parser as guessit_parser
 from sickrage.helper.common import remove_extension
 from sickrage.helper.encoding import ek
 from sickrage.helper.exceptions import ex
 from sickbeard.helpers import remove_non_release_groups
 import dateutil
 
+pre_cleanup_re = re.compile(r'(\.vol\d+\+\d+)?((\.par2\b)|(\.nzb\b)|(\.mkv\b)).*((\d+\.of\.\d+)|(\(\d+/\d+\))).*$',
+                            flags=re.IGNORECASE)
 
 class NameParser(object):
     ALL_REGEX = 0
```

Comment:
While your at it, and your the one who knows most about the parsing class, would you be willing to do some docstringing? Like here for the class, and for the class methods like the _parse_string() and parse() methods.